### PR TITLE
Add deprecation messages to `add win` and `remove win` functions

### PR DESCRIPTION
**Pull request by ALazyMeme**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
- [x] Documentation in docs/ or install-docs/ was updated, if applicable
- [x] I have tested all changes

### CODE REVIEW

**Review by pajlada — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ Remember to bring your dependencies up to date with `./scripts/venvinstall.sh` w
 - Bugfix: Fix errors in the API not properly returning a JSON response. (#1833)
 - Bugfix: Fix command "Check message" option not being modifiable. (#1845)
 - Bugfix: Work around no VIPs being refreshed through VIP refresh module in some cases. (#1862)
+- Dev: Add deprecation messages to `add_win` & `remove_win` functions.
```
Comment (RAnders00) [inline CHANGELOG.md:28]: Add PR number here please

Comment (pajlada) [reply CHANGELOG.md:28]: Will do when stable release is made! good catch

Comment (pajlada) [reply CHANGELOG.md:28]: Fixed here: https://github.com/pajbot/pajbot/pull/1942/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR32

Comment (RAnders00) [reply CHANGELOG.md:28]: Perfect! Thanks

**Pull request merged**